packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -142,9 +143,14 @@ async function getMendixVersion() {
         const dockerTagsResponse = await fetch(
             "https://registry.hub.docker.com/v1/repositories/mendix/runtime-base/tags"
         );
-        const runtimeVersions = (await dockerTagsResponse.json())
-            .filter(r => !r.name.startsWith("9"))
-            .map(r => r.name.split("-")[0]);
+
+        let dockerTagsJson = await dockerTagsResponse.json();
+
+        if (mendixMajorVersion) {
+            dockerTagsJson = dockerTagsJson.filter(r => r.name.startsWith(`${mendixMajorVersion}`));
+        }
+
+        const runtimeVersions = dockerTagsJson.map(r => r.name.split("-")[0]);
```

Comment:
Maybe use `r.name.split("-").shift()` instead? Not really necessary but we avoid usage of manual techniques.

Reply:
True, although it will do some extra work altering the array. I will change it to `shift()` though.